src/DynamoCoreWpf/ViewModels/Core/ConnectorViewModel.cs

```diff
@@ -1195,6 +1202,28 @@ private void nodeEndViewModel_PropertyChanged(object sender, System.ComponentMod
             }
         }
 
+        void StartPortModel_PropertyChanged(object sender, PropertyChangedEventArgs e)
+        {
+            switch (e.PropertyName)
+            {
+                case nameof(PortModel.Center):
+                    RaisePropertyChanged(nameof(CurvePoint0));
+                    Redraw();
+                    break;
+            }
+        }
+
+        void EndPortModel_PropertyChanged(object sender, PropertyChangedEventArgs e)
+        {
+            switch (e.PropertyName)
+            {
+                case nameof(PortModel.Center):
+                    RaisePropertyChanged(nameof(CurvePoint1));
```

Comment:
not sure if this is the right property for which to raise an event.
Does CurvePoint1 correspond to the end Node ?
I did see mentioned somewhere in the code that CurvePoint0 corresponds to the start Node.

Reply:
Looking at the Redraw function...it looks like CurvePoint3 should correspond to the end Node